schemas/link.ts

```diff
@@ -1,21 +1,26 @@
-export default {
+import { defineField, defineType } from "sanity";
+
+const link = defineType({
   name: "link",
   title: "Link",
   type: "object",
   fields: [
-    {
+    defineField({
       title: "Titulo/Nombre de dominio",
       name: "name",
       type: "string",
-    },
-    {
+    }),
+    defineField({
       title: "Enlace",
       name: "url",
       type: "url",
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       validation: (Rule: any) =>
```

Comment:
El comment ya no seria necesario (o el type `any`), `defineField` nos proporciona los types necesarios

```suggestion
      validation: (Rule) =>
```